packages/notifications/tests/unit/navi-notifications-test.js

```diff
@@ -0,0 +1,35 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+let Container, ServiceFactory;
+
+module('Unit | Service | navi-notifications', function(hooks) {
+  setupTest(hooks);
+
+  hooks.beforeEach(function() {
+    Container = this.owner;
+    ServiceFactory = Container.factoryFor('service:navi-notifications');
```

Comment:
```suggestion
    ServiceFactory = this.owner.factoryFor('service:navi-notifications');
```

Do we need to declare Container globally?